doc/sphinx/README.rst

```diff
@@ -97,7 +97,7 @@ Here is the list of all objects of the Coq domain (The symbol :black_nib: indica
 ``.. cmd::`` :black_nib: A Coq command.
     Example::
 
-       .. cmd:: Infix @string := @term1_extended {? ( {+, @syntax_modifier } ) } {? : @ident }
+       .. cmd:: Infix @string := @one_term {? ( {+, @syntax_modifier } ) } {? : @ident }
```

Comment:
You modified the `README` without modifying the template. This is probably what makes the dune build fail.

Reply:
This is not well documented.  It behaves differently from all the other .rst files.   `README.template.rst` only contains the string `[OBJECTS]` rather than this modified line.  I don't see where the value for `OBJECTS` comes from.  A global search for the string doesn't help.

The instructions in README.rst are inadequate:

```
README.rst is auto-generated from README.template.rst and the coqrst docs;
   use ``doc/tools/coqrst/regen_readme.py`` to rebuild it.
```

I don't know where to find the coqrst docs.  This should give the filenames.  And it should explain the OBJECTS and other substituted values.

Reply:
Ugh.  It's from the coqdomain.py, which certainly isn't a "coqrst doc" IMHO.  I updated the comment in README.rst.

Reply:
Anyway, the idea is that you shouldn't modify this file directly.

Reply:
It's funny because now you've updated the comment at the top saying that it was an auto-generated file instead of updating the comment in the template and regenerating the file.